mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOpTraits.h

```diff
@@ -64,6 +66,28 @@ class SYCLGetGroup : public OpTrait::TraitBase<ConcreteType, SYCLGetGroup> {
     return verifySYCLGetGroupTrait(Op);
   }
 };
+
+///  This interface describes an operation returning either a SYCL ID type (for
+///  cardinality 0) or an index type (for cardinality 1).
```

Comment:
I assume this is the MLIR builtin `index` type?

Reply:
Yep, as it is in the case in the `gpu` dialect operations.

Reply:
Maybe we can clarify this in the comment here and below.

Reply:
Done!